### DISCUSSION TITLE
[PSD6] Esercizio Slide 9

### DISCUSSION THREAD

**Discussion by matypist** [sapienzastudentsnetwork/psd2223#150] [Esercizi (A-L)]
Date le funzioni $x$, $y$ e $z$ definite dalla tabella di verità:

![image](https://user-images.githubusercontent.com/69389461/210017246-0808c178-4dd1-4ee1-8bd8-0755efb1f914.png)

- Scrivere $x$, $y$ e $z$ in forma canonica SOP e POS
- Semplificare le forme SOP

**Comment by Elia-Belli** [comment]
- **SOP canonica**:
x = $\bar A \bar B \bar C + \bar A B C + A \bar B C + A B \bar C$
y = $A \bar B \bar C + A B \bar C$
z = $\bar A \bar B \bar C + \bar A B \bar C + \bar A B C + A \bar B \bar C+ A \bar B C + A B \bar C$
- **SOP minima**:
x = $\bar A \bar B \bar C + \bar A B C + A \bar B C + A B \bar C$
y = $A\bar C$
z = $\bar A B + \bar C + A \bar B$
- **POS canonica**:
x = $(A+B+ \bar C) (A + \bar B + C)(\bar A + B + C)(\bar A + \bar B + \bar C)$
y = $(A + B + C) (A + B + \bar C)(A + \bar B + C)(A + \bar B + \bar C)(\bar A + B + \bar C)(\bar A + \bar B + \bar C)$
z = $(A + B + \bar C)(\bar A + \bar B + \bar C)$
- **POS minima**:
x = $(\overline{A}+\overline{B}+\overline{C})(\overline{A}+C+B)(\overline{B}+C+A)(\overline{C}+B+ A)$
y = $A\bar C$
z = $(\bar A + \bar B + \bar C)(A + B +\bar C)$

**Comment by FeddyLix17** [comment]
tutto corretto
